std/special/compiler_rt.zig

```diff
@@ -9,7 +9,7 @@ comptime {
     @export("__getf2", @import("compiler_rt/comparetf2.zig").__getf2, linkage);
 
     if (!is_test) {
-        // only create these aliases when not testing
+        // Why don't we need these when testing?
```

Comment:
I don't remember, to be honest. You can delete my unhelpful comment (apologies for that). I don't think changing it to a question helps anything though.

Reply:
I figured it out. These are aliases, not the canonical names...I will update this comment